src/coreclr/pal/tests/palsuite/common/palsuite.cpp

```diff
@@ -53,13 +54,7 @@ char* convertC(const WCHAR * wString)
 
 UINT64 GetHighPrecisionTimeStamp(LARGE_INTEGER performanceFrequency)
 {
-    LARGE_INTEGER ts;
-    if (!QueryPerformanceCounter(&ts))
-    {
-        Fail("ERROR: Unable to query performance counter!\n");      
-    }
-    
-    return ts.QuadPart / (performanceFrequency.QuadPart / 1000);    
+    return (UINT64)minipal_hires_ticks();
 }
```

Comment:
This is a very sloppy change :(

Reply:
It should be `minipal_lowres_ticks` indeed because it only uses ms precision.